server/src/main/java/org/spine3/server/stand/AggregateStateIdStringifier.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.stand;
+
+import com.google.common.base.Enums;
+import com.google.common.base.Optional;
+import com.google.protobuf.Message;
+import org.spine3.base.Stringifier;
+import org.spine3.base.Stringifiers;
+import org.spine3.type.ClassName;
+import org.spine3.type.KnownTypes;
+import org.spine3.type.TypeName;
+import org.spine3.type.TypeUrl;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.spine3.base.Identifiers.checkSupported;
+import static org.spine3.util.Exceptions.wrappedCause;
+
+/**
+ * A {@link Stringifier} for the {@link AggregateStateId}.
+ *
+ * <p>The protocol of strigification is as follows:
+ * {@code {@linkplain TypeUrl}_of_the_state_type}-{@code ID_TYPE_NAME_or_SHORT_REPR_FOR_PRIMITIVES}-{@code generic_ID_stringifier_output}
```

Comment:
Can we please make it more readable? A short sample usage instead of this line would do.